docs/development-guide/DevelopmentGuide.md

```diff
@@ -118,11 +118,20 @@ For more details on failure detection settings, see [here](../using-the-jdbc-dri
 For more details on failover settings, see [here](../using-the-jdbc-driver/FailoverConfigurationGuide.md).
 ![](../images/jdbc_wrapper_postgresql_failover_efm_30000_5000_3.png)
 ![](../images/jdbc_wrapper_postgresql_failover_efm_6000_1000_1.png)
-See [here](PerformanceResults.md#failover-performance-with-different-enhanced-failure-monitoring-configuration) for a more detailed performance breakdown.
+See [here](EFMAndFailoverPluginPerformanceResults.md#failover-performance-with-different-enhanced-failure-monitoring-configuration) for a more detailed performance breakdown.
 
 ![](../images/jdbc_wrapper_postgresql_efm_30000_5000_3.png)
 ![](../images/jdbc_wrapper_postgresql_efm_6000_1000_1.png)
-See [here](PerformanceResults.md#enhanced-failure-monitoring-performance-with-different-failure-detection-configuration) for a more detailed performance breakdown.
+See [here](EFMAndFailoverPluginPerformanceResults.md#enhanced-failure-monitoring-performance-with-different-failure-detection-configuration) for a more detailed performance breakdown.
+
+#### Read-Write Splitting and Reader Load Balancing Performance Tests
+
+The diagrams in this section show the AWS JDBC Driver's performance when using the read-write splitting plugin, with or without reader load balancing. This test sets up a large number of connections in parallel; the initial connection is to the writer but will be changed to various reader instances if the plugin is enabled. The test executes a long query many times to simulate heavy queries.
+
+The average overhead time is measured as the average time difference between running the test when no plugins are enabled and running the test with the plugins enabled. The baseline number indicates there is no overhead time when no plugins are enabled.
```

Comment:
Reworded for a bit more clarity: `The average overhead time is measured as the average time difference between running the test with the read-write plugin and running the test without any plugins. The baseline overhead time is 0 because there are no plugins in this scenario and thus there is no plugin overhead.`